### ISSUE TITLE
1281 - Subtract the Product and Sum of Digits of an Integer

### ISSUE THREAD

**Issue by AN4553R** [vJechsmayr/PythonAlgorithms#640]
## Description of the Problem
Given an integer number `n`, return the difference between the product of its digits and the sum of its digits. 

## Code
```
class Solution:
    def subtractProductAndSum(self, n: int) -> int:
```

## Link To The LeetCode Problem
[LeetCode](https://leetcode.com/problems/subtract-the-product-and-sum-of-digits-of-an-integer/)

**Comment by AN4553R** [comment]
Please assign me to this

**Comment by AN4553R** [comment]
Solved in #638
